app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -92,6 +92,7 @@ public class ContributionsFragment
     private static final String CONTRIBUTION_LIST_FRAGMENT_TAG = "ContributionListFragmentTag";
     private MediaDetailPagerFragment mediaDetailPagerFragment;
     static final String MEDIA_DETAIL_PAGER_FRAGMENT_TAG = "MediaDetailFragmentTag";
+    private static final int MAX_RETRIES = 10;
```

Comment:
Have we considered the following points to decide the max retry count?

- High retry count can put an increased load on servers, especially if multiple users are retrying simultaneously, potentially affecting overall service performance.
- High consumption of battery, network bandwidth, and device processing power, impacting device performance and battery life.
- In case of a low retry count, some tasks might not complete successfully due to transient issues that could have been resolved with more retries.

Reply:
I think 10 is reasonable.

Reply:
Yes 10 seems fine. I just wanted to ensure if we've weighed all the pros and cons to decide the retry count.